go.mod

```diff
@@ -5,10 +5,9 @@ go 1.15
 require (
 	contrib.go.opencensus.io/exporter/zipkin v0.1.2
 	github.com/gin-gonic/gin v1.7.2
-	github.com/go-logr/stdr v1.2.2 // indirect
```

Comment:
Huh. I was able to remove this one without even getting rid of the logging exporter.

Reply:
The logging exporter still works.